Assets/Scripts/Checkpoint.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Checkpoint : MonoBehaviour {
+
+
+    //variable that needs to be applied within unity
+    [SerializeField]
+    private float inactivatedScale = 1, activatedScale = 1.5f;
```

Comment:
Either assign the variables in the Start Function or serialize the field. I recommend not serializing a field.